plugins/env/payloadschema.go

```diff
@@ -0,0 +1,40 @@
+// Originally generated by github.com/taskcluster/jsonschema2go
+// But as jsonschema2go doesn't handle addtionalProperties smoothly,
+// we manually modified it (I apologize the world for that).
+
+package env
+
+import (
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+type (
+	// env payload section contents
+	envPayload map[string]string
+)
+
+var envPayloadSchema = func() runtime.CompositeSchema {
+	schema, err := runtime.NewCompositeSchema(
+		"env",
+		`
+		{
+		  "$schema": "http://json-schema.org/draft-04/schema#",
+		  "additionalProperties": {
+		    "description": "Each key/value pair represents an environment variable configuration",
+		    "type": "string"
+		  },
+		  "description": "env payload section contents",
+		  "title": "EnvPayload",
+		  "type": "object"
+		}
+		`,
+		false,
+		func() interface{} {
+			return &envPayload{}
```

Comment:
Does this build?
I would suggest `make(map[string]string)` and not define a type `envPayload`